bin/sandbox

```diff
@@ -53,10 +53,10 @@ fi
 cd ./sandbox
 cat <<RUBY >> Gemfile
 
+gem 'awesome_nested_set', github: 'peterberkenbosch/awesome_nested_set', branch: 'rails-7-support'
 gem 'solidus', path: '..'
 gem 'solidus_auth_devise', '>= 2.1.0'
-gem 'rails-i18n'
-gem 'solidus_i18n'
+gem 'solidus_i18n', github: 'solidusio/solidus_i18n'
```

Comment:
Do we need to release a new version of solidus_i18n with the code currently in master?

Reply:
I don't think so, revisiting this now. 2.1.1 should have the latest from master now anyway right?

Reply:
Confirming we don't need to do this anymore. There are a few deprecation warnings showing up, looking into those as well, but it's working as expected right now.